test/integration/nodeagent/lease/lease_test.go

```diff
@@ -0,0 +1,116 @@
+// Copyright 2023 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package lease_test
+
+import (
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	coordinationv1 "k8s.io/api/coordination/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/utils/pointer"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	. "github.com/gardener/gardener/pkg/utils/test/matchers"
+)
+
+var _ = Describe("Reconcile", func() {
+	Describe("Lease controller tests", func() {
+		var (
+			node *corev1.Node
+		)
+
+		BeforeEach(func() {
+			node = &corev1.Node{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:   nodeName,
+					Labels: map[string]string{testID: testRunID},
+				},
+			}
+			lease := &coordinationv1.Lease{ObjectMeta: metav1.ObjectMeta{Namespace: testNamespace.Name, Name: "gardener-node-agent-" + nodeName}}
+
+			By("Create Node")
+			Expect(testClient.Create(ctx, node)).To(Succeed())
+			DeferCleanup(func() {
+				By("Delete Node")
+				Expect(client.IgnoreNotFound(testClient.Delete(ctx, node))).To(Succeed())
+				By("Delete Lease")
+				Expect(client.IgnoreNotFound(testClient.Delete(ctx, lease))).To(Succeed())
+			})
+		})
+
+		It("should create the Lease", func() {
+			lease := &coordinationv1.Lease{}
+			Eventually(func() error {
+				return testClient.Get(ctx, types.NamespacedName{Namespace: testNamespace.Name, Name: "gardener-node-agent-" + nodeName}, lease)
+			}).Should(Succeed())
+			validateOwnerReference(lease, node)
+		})
+
+		It("should update the Lease time", func() {
+			lease := &coordinationv1.Lease{}
+			Eventually(func() error {
+				return testClient.Get(ctx, types.NamespacedName{Namespace: testNamespace.Name, Name: "gardener-node-agent-" + nodeName}, lease)
+			}).Should(Succeed())
+			validateOwnerReference(lease, node)
+
+			oldRenewTime := lease.Spec.RenewTime
+			// wait a nit more than LeaseDurationSeconds
+			fakeClock.Step(5 * time.Second)
```

Comment:
Can you store the `LeaseDurationSeconds` in a variable and use it here and in other occurrences? This way, when somebody changes the value, they wouldn't need to adapt all these places here as well.